debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/ModularityHelper.java

```diff
@@ -97,7 +97,7 @@ public static boolean moduleFilterMatchAll(J9ModulePointer modulePtr, String arg
 	 *                      be outputted to.
 	 */
 	public static void printJ9Module(J9ModulePointer modulePtr, PrintStream out) throws CorruptDataException {
-		String moduleName = J9ObjectHelper.stringValue(modulePtr.moduleName());
+		String moduleName = J9UTF8Helper.stringValue(modulePtr.moduleName());
```

Comment:
Changes like this need to be sensitive to the vintage of the JVM that created the core file (it will still be `j9object_` in older core files).

Reply:
Right. I'll think reconsider how we should handle this.

Reply:
We can add a new helper method in `ModularityHelper` to retrieve the module name, which can then be called elsewhere as needed.
```
String getModuleName(J9ModulePointer modulePtr) {
    String moduleName = null;
    if (modulePtr != null) {
        Object mn = modulePtr.moduleName();
        if (mn instanceof J9ObjectPointer) {
            moduleName = J9ObjectHelper.stringValue(mn);
        } else if (mn instanceof J9UTF8Pointer) {
            moduleName = J9UTF8Helper.stringValue(mn);
        } 
    }
    return moduleName;
}
```